examples/mcp-client/src/server.ts

```diff
@@ -0,0 +1,85 @@
+import {
+  Agent,
+  routeAgentRequest,
+  type AgentNamespace,
+  type Connection,
+} from "agents";
+
+type Env = {
+  MCP_SERVERS: { host: string; info: { name: string; version: string } }[];
```

Comment:
listChanged as vNext cc @dinasaur404 since we have been talking to people asking if we'd support

Reply:
https://spec.modelcontextprotocol.io/specification/2024-11-05/basic/lifecycle/